### DISCUSSION TITLE
CE certified

### DISCUSSION THREAD

**Discussion by bkbartk** [arendst/Tasmota#13718] [Q&A] [answered]
Hello,

I'm not sure if blakadder is an official Tasmota site or nog.
But I was wondering is all EU products are CE certified or not?
I am looking at this powerplug
https://templates.blakadder.com/athom_PG01EU16A.html
but I actually desire for all my electric equipment to be CE certified. This gives me some feeling of safety.

**Comment by barbudor** [comment, marked as answer]
Yes, China Export
https://en.wikipedia.org/wiki/CE_marking#%22China_Export%22

If you want real CE certified products, get Shelly : designed in the EU 👍

**Reply by sfromis** [reply]
CE is not a certification, but the manufacturer can simply declare conformity. They *might* seek an independent assessment, or not. National authorities *might* ask them to provide documentation. They may get slapped in case of clear non-conformity, but meaningful enforcement is hard from outside the country of manufacturing. "China Export" is just an urban myth, as mentioned in the link from barbudor.

**Comment by Jason2866** [comment]
"Feeling of safety" CE is nothing more than this. If you want checked safety the device needs a real certificate. For example VDE https://www2.vde.com/en/Institute/Portfolio/Certification/Pages/VDE_Certification_Marks.aspx

**Comment by ascillato** [comment]
If you want safety, you have to protect your whole home with breakers, proper grounding, ground leak breakers, proper wire and loads calculations, fire detectors, etc. Also, it is nice to have a home insurance (those are really cheap in most countries). Some insurance companies will require that you present the UL certification of the devices installed. Shelly has UL certification.

FYI:
> I'm not sure if blakadder is an official Tasmota site or not.

yes it is.

**Comment by bkbartk** [comment]
I know CE isn't 100% safe as manufacturers are allowed to certify themselves. But it at least makes sure some guidelines are followed.
Also I'm aware of Chine Export stuff. 
And I'm not sure what is better, UL or VDE. But when it comes to electic stuff it should at least be a little safe. 
Fire detectors and insurance are nice and I have them. But i do not intent to use them.

But from what I'm getting is that I need to buy the Shelly. I already took a look at that one, but didn't like the led ring.
Now I'm convinced I need this one anyway.

Thank you all for responding

**Reply by sfromis** [reply]
No, "CE" does not *make sure* that any guidelines are followed, it is merely a claim that they have been, without any realistic avenues for enforcement in case of transgressions even being found out. If anything is certified "for real", you should be able to obtain certification documents from the **independent** entity performing the certification.
